### PR TITLE
Add parquet support and make memory more efficient

### PR DESCRIPTION
**Pull request by erykoff**
This PR sits on top of Erin's boolean and nside branch, and adds more fixes that were used to generate the 131k boolean map and associated 16k and 4k fracdet maps.

A number of memory tweaks were done, though it still uses a ton of memory!  Also this needs a bugfix in healsparse on writing high density parquet files.

### CODE REVIEW

**Comment by esheldon** [comment]
I think you also ran `pcm-make-mdet-mask` with `--use-bool` right?

**Comment by erykoff** [comment]
I must have ...

**Comment by esheldon** [comment]
Can you please  put that command in the README as well?

File: bin/pcm-make-mdet-mask-lowmem [hunk 1]
```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python
```
Comment (esheldon) [inline bin/pcm-make-mdet-mask-lowmem]: Should we delete this file?

**Comment by esheldon** [comment]
I want to rework the README but I'll do that separately

**Review by esheldon — APPROVED** [review]

**Pull request merged**